### PR TITLE
lab 3 notes

### PR DESCRIPTION
**Pull request by rmartin-sc**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,24 +42,24 @@ <h1>Welcome to Liam.L's CSD120 Website!</h1>
...
-                    <tr><th rowspan = "5">Winter</th><td>CSA103</td><td>Business Application</td><td>4</td></tr>
+                    <tr><th rowspan="5" scope="rowgroup">Winter</th><td>CSA103</td><td>Business Application</td><td>4</td></tr>
                     <tr><td>CSD102</td><td>Programming Using C++</td><td>5</td></tr>
                     <tr><td>CSD212</td><td>Web Scripting Languages</td><td>4</td></tr>
                     <tr><td>CSD102</td><td>Introduction to Linux</td><td>4</td></tr>
                     <tr><td>CST104</td><td>PC Hardware and Networking</td><td>5</td></tr>
                     </tbody>
                     <tfoot>
-                    <tr><th colspan = "3">Total</th><td>41</td></tr>
+                    <tr><th colspan="3" scope="row">Total</th><td>41</td></tr>
```
Comment (rmartin-sc) [inline index.html:62]: Missing scope attributes
Convention is to have no spaces around attribute equal signs

File: survey.html [hunk 2]
```diff
@@ -81,23 +81,22 @@ <h1>Survey</h1>
             >
             <input
               id="email"
-              type="text"
+              type="email"
```
Comment (rmartin-sc) [inline survey.html:84]: Wrong type

File: survey.html [hunk 3]
```diff
@@ -81,23 +81,22 @@ <h1>Survey</h1>
...
+            <input id="number" type="number" name="best_whole_number" min="1" step="1" />
           </p>
           <p>
             <label for="date">Which date is best?:</label>
             <input
               id="date"
               type="date"
-              name="best_date"
-              value="yyyy-mm-dd"
+              name="best_day"
```
Comment (rmartin-sc) [inline survey.html:99]: Were you trying to make a placeholder?

File: survey.html [hunk 4]
```diff
@@ -125,7 +124,7 @@ <h1>Survey</h1>
                 <input
                   type="radio"
                   name="answer"
-                  value="Yes"
+                  value="yes"
```
Comment (rmartin-sc) [inline survey.html:127]: Just correcting some typos to make grading work better

**Review by liamSL — APPROVED** [review]

**Pull request merged**